public/pages/utils/anomalyResultUtils.ts

```diff
@@ -95,7 +95,11 @@ const sampleMaxAnomalyGrade = (anomalies: any[]): any[] => {
   return sampledAnomalies;
 };
 
-export const prepareDataForChart = (data: any[], dateRange: DateRange) => {
+export const prepareDataForChart = (
+  data: any[],
+  dateRange: DateRange,
+  getFloorPlotTime?: any
```

Comment:
maybe name it "normalizeFunc"?

Reply:
Thanks for advice. It's not only normalize the timestamp to floor time in this function. It has filtering and sampling. So prefer to keep this name.